setup-scripts/health-checker.py

```diff
@@ -114,55 +114,86 @@ async def check_health():
-        res = requests.get("http://localhost/health-check", verify=False)
+        api = "http://localhost"
+        res_check = requests.get(api + "/health-check", verify=False)
+        json_check = res_check.json()
+        json_critical = requests.get(api + "/health-check/critical", verify=False).json()
+        json_verbose = requests.get(api + "/health-check/verbose", verify=False).json()
     except:
         trace = traceback.format_exc()
         print("[DEBUG] check_health() failed.")
-        if len(trace) < DISCORD_MAX_MESSAGE_LENGTH:
-            await send_msg(client, "```\n{}\n```".format(trace), force_notify=False)
-        else:
-            await send_msg(client, "Failed to run the checks!",
-                           file=discord.File(io.BytesIO(trace.encode()), filename="failed_checks.log"),
-                           force_notify=True)
-        return
+        return await send_msg(client, "Failed to run the checks!", file=trace, force_notify=True)
+
+    critical_checks_total = 0
+    critical_checks_failed = 0
+
+    verbose_checks_total = 0
+    verbose_checks_failed = 0
 
-    # Check the health records.
-    passed_checks = 0
-    failed_checks = 0
-    failed_critical = 0
     failed_records = []
+    failed_records_file = None
+
     time_limit_unaware = datetime.now() - timedelta(hours=CHECK_HOURS)  # local time
     time_limit = time_limit_unaware.astimezone(get_localzone())  # time with time zone
-    for rec in res.json():
-        time_unaware = datetime.strptime(rec['date'], '%Y-%m-%dT%H:%M:%S.%fZ')  # time in UTC
+
+    for critical in json_critical:
+        time_unaware = datetime.strptime(critical['date'], '%Y-%m-%dT%H:%M:%S.%fZ')  # time in UTC
         time = pytz.utc.localize(time_unaware)  # time with time zone
         if time < time_limit:
             continue
         bad = False
-        for check in rec['checks']:
+        for check in critical['checks']:
+            critical_checks_total += 1
             if check['up'] == False:
+                critical_checks_failed += 1
                 bad = True
-                failed_checks += 1
-                if check['critical']:
-                    failed_critical += 1
         if bad:
-            # We append the entire record, so we can get the full context.
-            failed_records.append(rec)
-        passed_checks += 1
-
-    checks = passed_checks + failed_checks
-    if len(failed_records) > 0:
-        message = "Found {}/{} failed checks ({} critical) over the last {} hours!".format(failed_checks, checks,
-                                                                                           failed_critical, CHECK_HOURS)
-        file = discord.File(io.BytesIO(json.dumps(failed_records, indent=2).encode()), filename="failed_checks.log")
-        notifyTeam = failed_critical > 0
-        await send_msg(client, message, file=file, force_notify=notifyTeam)
-        return
+            failed_records.append(critical["checks"])
```

Comment:
Let's appends the entire `critical` here - dropping the date won't help us much but it might prevent us from seeing a connection with something else.